pandas/tests/copy_view/test_methods.py

```diff
@@ -1117,6 +1117,29 @@ def test_putmask(using_copy_on_write):
         assert view.iloc[0, 0] == 5
 
 
+def test_asfreq_noop(using_copy_on_write):
+    df = DataFrame(
+        {
+            "a": Series(
+                [0.0, None, 2.0, 3.0], index=date_range("1/1/2000", periods=4, freq="T")
+            )
+        }
```

Comment:
```suggestion
        {"a": [0.0, None, 2.0, 3.0]},
        index=date_range("1/1/2000", periods=4, freq="T"),
```

In general I think it's best to avoid constructions from other DataFrame/Series objects if not needed. Because of being constructed by a Series, that could trigger a CoW when mutating the parent (although in theory the temporary Series should already be cleaned up), and if we want to assert that mutating the parent triggers CoW, we might be accidentally not checking exactly what was intended (even if `df2` didn't properly keep track of refs to `df`, mutating `df` could still trigger CoW and not update `df2` because of the construction)

And it also just makes the formatting simpler in this case ;)

Reply:
Thx, I just copied the example from the docstring of asfreq, probably wasn't a great idea :)